src/gui/basegraphgl.cpp

```diff
@@ -426,19 +462,77 @@ void BaseGraphGL::resizeEvent(QResizeEvent* e)
     updateCache();
 }
 
+void BaseGraphGL::updateEdgeInspector(const Edge& edge)
+{
+    m_ui->inspector->setCurrentIndex(1);
+    
+    m_ui->edgeId->setText(QString::number(edge.id()));
+    
+    m_ui->inspector->show();
+    m_ui->inspector->adjustSize();
+    m_inspGeo = m_ui->inspector->frameGeometry();
+    m_inspGeo += QMargins(5,5,5,5);
+}
+
+void BaseGraphGL::updateEdgesInspector(const Node& pnode, const Node& cnode)
+{
+    m_ui->edgesList->clear();
+    QSet<int> edges;
+    
+    for (auto const& e : pnode.outEdges()) {
+        if (e.second.neighbour().id() == cnode.id()){
+            edges.insert(e.first);
+        }
+    }
+    
+    if (edges.size() == 0){
+            return;
+    }
```

Comment:
I'm not sure it would produce the same results. ``pnode.outDegree()`` would test for all edges whereas we want to test whether there exists an edge connecting the two specific nodes.

Reply:
oh yeah, sorry; you're right ;)